genai-on-vertex-ai/genai-website-mod/backend/config.toml

```diff
@@ -1,6 +1,6 @@
 [global]
-project_id = ""
-location = ""
+project_id = "kalschi-csm-5"
```

Comment:
Let's scrub the project id as its internal only

Reply:
removed